packages/amplify-graphql-index-transformer/src/graphql-index-transformer.ts

```diff
@@ -19,7 +19,7 @@ import {
   ObjectTypeDefinitionNode,
 } from 'graphql';
 import { isListType, isScalarOrEnum } from 'graphql-transformer-common';
-import { appendSecondaryIndex, constructSyncVTL, updateResolversForIndex, getResourceOverrides, getDeltaSyncTableTtl } from './resolvers';
+import { appendSecondaryIndex, constructSyncVTL, updateResolversForIndex, getResourceOverrides, getDeltaSyncTableTtl } from './resolvers/resolvers';
```

Comment:
You should still be able to do imports without the lower resolution as long as the right lines are added to the index.ts there